api/src/controllers/animeController.ts

```diff
@@ -0,0 +1,82 @@
+import type { Request, Response, NextFunction } from '@@types/express.js';
+import { errors, entities } from '@@utils/index.js';
+import { Anime } from '@@entities/index.js';
+
+interface Params {
+    animeId: number
```

Comment:
This can just be `id` so when we outline it in `/src/routes` we can have it as `/anime/id/:id`